lib/src/utils.dart

```diff
@@ -50,3 +50,11 @@ Future<void> saveLyricsOfMultipleSongs(
         verbose: verbose);
   }
 }
+
+dynamic firstWhereOrNull(
```

Comment:
why did you write this method ?  i think dart already do something like [this](https://api.flutter.dev/flutter/package-collection_collection/IterableExtension/firstWhereOrNull.html)

Reply:
That's from a package called `collection, right?

Reply:
yes, why? 🤔

Reply:
I thought that was an external package😅